lib/chef/win32/api/file.rb

```diff
@@ -565,6 +600,21 @@ def retrieve_file_info(file_name)
           file_information
         end
 
+        def retrieve_file_version_info(file_name)
+          file_name = encode_path(file_name)
+          file_size = GetFileVersionInfoSizeW(file_name, "")
```

Comment:
yeah. "A pointer to a variable that the function sets to zero." We don't do anything with it. Is there a potential issue with passing an empty string instead of creating a pointer variable?